ores/scoring_systems/celery_queue.py

```diff
@@ -161,14 +182,15 @@ def _lookup_inprogress_results(self, request, response):
                    model_name in response.scores[rev_id]:
                     continue
 
+                if not self.jobs_lock_manager:
+                    continue
                 task_id = context.format_id_string(
                     model_name, rev_id, request,
                     injection_cache=injection_cache)
-                score_result = \
-                    self._lookup_score_in_map.AsyncResult(task_id)
-                if score_result.state in (REQUESTED, SENT,
-                                          celery.states.STARTED,
-                                          celery.states.SUCCESS):
+                lock_value = self.jobs_lock_manager.is_locked(task_id)
```

Comment:
Method should be something like 
```
group_task_id = self.score_deduplicator.has_in_progress_task(model_task_id)
...
score_result = self._process_score_map.AsyncResult(group_task_id)
```

Reply:
I renamed the method to get_in_progress_task but I don't want to change it to anything beside a string key, We might want to track and deduplicate using different keys, all logic related to how to turn objects to a key should stay in this class IMO. Basically similar to cache objects, we should just send keys and not objects.